provider/aws_sd.go

```diff
@@ -667,5 +674,8 @@ func (p *AWSSDProvider) serviceTypeFromEndpoint(ep *endpoint.Endpoint) string {
 
 // determine if a given hostname belongs to an AWS load balancer
 func (p *AWSSDProvider) isAWSLoadBalancer(hostname string) bool {
```

Comment:
Does is make sense to have seperate functions (Having one for ELB and another one for NLB)?

Reply:
I don't think so. We actually want to treat both types of load-balancers the same.